cryptlib.vcxproj

```diff
@@ -126,7 +126,7 @@
       <InlineFunctionExpansion>AnySuitable</InlineFunctionExpansion>
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <StringPooling>true</StringPooling>
-      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
+      <RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>
     </ClCompile>
     <ResourceCompile>

```

Comment:
i would like to also see something like this
```
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">^M
+    <Lib>^M
+      <OutputFile>$(OutDir)$(TargetName)d$(TargetExt)</OutputFile>^M
+    </Lib>^M
+  </ItemDefinitionGroup>^M
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">^M
+    <Lib>^M
+      <OutputFile>$(OutDir)$(TargetName)d$(TargetExt)</OutputFile>^M
+    </Lib>^M
+  </ItemDefinitionGroup>^M
``` 

and not to compose the resulting filename manually in our cmake-script

Reply:
done